test/unit-tests/config.test.ts

```diff
@@ -21,9 +21,11 @@ function createConfig(conf: Partial<ExtensionConfigurationSettings>): Configurat
         buildArgs: [],
         buildToolArgs: [],
         parallelJobs: 0,
+        allowParallelJobs: false,
```

Comment:
Does this setting exist?  I thought you only added `ctest.allowParallelJobs`

Reply:
yes, thanks for catching me on that!

Reply:
fixed